test_cursor_pr_review.py

```
@@ -18,7 +18,8 @@
 sys.path.insert(0, os.path.dirname(__file__))
 from cursor_pr_review import (
     ReviewConfig, ConfigError, APIError, SecurityError,
-    load_config, save_config, APIClient, create_github_workflow
+    load_config, save_config, APIClient, create_github_workflow,
+    save_github_workflow, review_pr
```

Ruff (0.11.9)

`cursor_pr_review.review_pr` imported but unused

Remove unused import: `cursor_pr_review.review_pr`

(F401)